README.md

```diff
@@ -288,6 +288,93 @@ Once Keycloak is up and running, login to the [Management Console](http://www.ke
 
 2. Choose the [sample realm](./alfresco-realm.json) file and click the "Create" button.
 
+## Upgrading from Identity Service 1.1 to 1.2
+
+  **_NOTE:_** The upgrade of the Alfresco Identity Management Service requires downtime. 
+  This means that no user will be able to connect to any of the Digital Business Platform components while the upgrade or rollback is being done.
+
+### General upgrade procedure
+
+For upgrading Alfresco Identity Management Service we are mainly following the Keycloak upgrade procedure.
+We will be explaining how to do it if you are using our out of the box distribution or Kubernetes deployment.
+However depending on the environment you are using you should follow the next steps:
+
+1. Prior to applying the upgrade, handle any open transactions and delete the data/tx-object-store/ transaction directory.
```

Comment:
Should we add any additional detail on how to handle open transactions or link to the Keycloak docs on that?

Reply:
I added in a link to user session management page.